pkgs/by-name/co/comictagger/package.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, unrar
+, python3
+, fetchgit
+, poetry
+, git
+, rar
+, qt5
+, python3Packages
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "comictagger";
+  version = "1.5.5";
+  format = "pyproject";
+
+  src = fetchgit {
```

Comment:
can you use `fetchFromGitHub` ?

Reply:
Hi, it will not work with fetch from github - it fetches the repo, but later will not compiling, requesting to use the what leaveDotGit = "true"; achieves.